portfolio/src/main/webapp/style.css

```diff
@@ -8,13 +8,13 @@
   width: 650px;
 }
 
-#greeting-container {
+#quote-container {
   margin-top: 20px;
-  font-family: Arial;
+  /* font-family: Arial; */
```

Comment:
To keep the code clutter to a minimum, avoid leaving commented out code after debugging.